libraries/Wire/src/utility/xmc_i2c_conf.h

```diff
@@ -64,10 +64,6 @@ extern XMC_I2C_t XMC_I2C_1;
 #define NUM_I2C  1
 extern XMC_I2C_t XMC_I2C_0;
 
-#elif defined(XMC1400_Arduino_Kit)
```

Comment:
Same as [xmc_spi_conf.c](https://github.com/Infineon/XMC-for-Arduino/pull/195/files#diff-f0a5b6e4a58530712f1cbce9463eed9daedae46d0f0acb1f8fde3641bd66ed4c)